deploy/monai-deploy-express/README.md

```diff
@@ -166,12 +179,17 @@ This example uses the `alpine` container image instead of a MAP to print all fil
 In this section, we will download a DICOM dataset, upload it to Orthanc and then run the [Liver Segmentation MAP](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/examples/apps/ai_livertumor_seg_app) from the
 [MONAI Deploy App SDK](https://github.com/Project-MONAI/monai-deploy-app-sdk). Finally, we can expect the AI-generated segmentation result to appear in Orthanc.
 
+
 #### Steps
 
 1. Download the [Abdomen CT dataset](#running-a-monai-deploy-workflow) dataset
-2. Upload the dataset as described in [Uploading Data](#upload-dicom-datasets)
-3. Deploy the workflow definition to MONAI Deploy Workflow Manager:
-4. 
+2. Download the AI Liver Tumor MAP
```

Comment:
NIT: Since we have this (pre) pull step, would be better have a step to pull the Lung seg image, too. Anyway, the second one does not take as long.

Reply:
See line 223